pages/checkout.vue

```diff
@@ -244,10 +261,30 @@ export default {
       return { ...checkout?.value }
     })
 
+    const numberOfItems = computed(() => checkoutGetters.getLineItemTotal(getOrder.value))
+    const cartSubTotal = computed(() => checkoutGetters.getSubtotal(getOrder.value))
+    const standardShipping = computed(() => checkoutGetters.getShippingTotal(getOrder.value))
+    const estimatedTax = computed(() => checkoutGetters.getTaxTotal(getOrder.value))
+    const estimatedOrderTotal = computed(() => checkoutGetters.getTotal(getOrder.value))
+
     const logIn = () => {
       toggleLoginModal()
     }
 
+    // coupons
+    // TODO
+    // const applyPromocode = async (couponApplied) => await applyCoupon(couponApplied)
+
+    const isValidCoupon = computed(() => !cart.value?.invalidCoupons[0]?.couponCode)
+
+    const invalidCouponErrorText = computed(
+      () => `${cart.value?.invalidCoupons[0]?.couponCode} ${context.root.$t("is an invalid code")}`
+    )
+
+    const appliedCoupons = computed(() => cart.value?.couponCodes)
+
+    const areCouponsApplied = computed(() => cart.value?.couponCodes.length > 0)
```

Comment:
several references to cart.value couponCodes. Since this is checkout page, they need to be using checkout/order object not cart

Reply:
Done